doc.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+
+repo=$1
+gitUrl=$2
+commitId=$3
+localPath=$4
+sidebarConfigFile=$5
+
+
+if [ ! -d "./tmp" ]; then
+  mkdir ./tmp
+fi
+cd ./tmp
+
+if [ ! -d "./${repo}" ]; then
+  git clone ${gitUrl}
+  cd ./${repo}
+  else
+  cd ./${repo}
+  git pull origin master
+fi
+
+git reset --hard ${commitId}
+
+if [ -d "../../${localPath}" ]; then
+  rm -rf ../../${localPath}
+fi
+mkdir -p ../../${localPath}
```

Comment:
Hi, just for sure, this `localPath` is used to store every project's docs?

After pull & move those docs to that folder, do we need to make some changes to them, so after building, every project's docs would have a universal view-feeling?

Reply:
> Hi, just for sure, this localPath is used to store every project's docs?

They will. Every repo will have `name`, `repo`, `version`, `commit id`. Then, all path are generated automatically based on rules.

Reply:
Last preview is available here, https://deploy-preview-215--hopeful-fermat-fec5f0.netlify.app/

![image](https://user-images.githubusercontent.com/5441976/107875404-93791f00-6efa-11eb-8bc7-f43655d14d92.png)

These are available for preview.

Reply:
Thanks! I will take a look :)